public/create_team.php

```diff
@@ -0,0 +1,80 @@
+<?php require_once(dirname(__FILE__)."/../modules/ensure_logged_in.php"); ?>
+<?php include "templates/header.php"; ?>
+
+<?php if(get_current_role() == 'instructor') {
+
+require_once "../modules/models/user.php";
+require_once "../modules/models/course_section.php";
+require_once "../modules/models/course_section_student.php";
+require_once "../common.php";
+
+$course_id = $_GET['id'];
+
+try {
+    $course_sections = CourseSection::where(array('offering_id' => $course_id));
+
+} catch (PDOException $error) {
+    echo "<br>" . $error->getMessage();
+}
+
+$course_students = array();
+
+foreach ($course_sections as $section) {
+    try {
+        $course_section_student = CourseSectionStudent::includes("user")->where(array('section_id' => $section->id));
```

Comment:
this is also weird, why don't you just include all this stuff directly in the CourseSection query

Reply:
done